cohorts/cohort.py

```diff
@@ -1088,9 +1099,11 @@ def get_filtered_isovar_epitopes(self, epitopes, ic50_cutoff):
         return EpitopeCollection(mutant_binding_predictions)
 
     def load_single_patient_isovar(self, patient, variants, epitope_lengths):
-        # TODO: different epitope lengths, and other parameters, should result in
-        # different caches
-        isovar_cached_file_name = "%s-isovar.csv" % self.merge_type
+        # different cache depending on epitope length & variant set
+        variant_hash = make_hash(frozenset(variants))
```

Comment:
`md5` only works for strings -- not Variants. Unless you `pickle.dump` them, or do `str(x)` for each -- which is also kind of hacky.